src/contributors/Upama677.md

```diff
@@ -0,0 +1,6 @@
+name: "Upama Duuta"
+username: "Upama677"
+image: "upama.jpg"

```

Comment:
- You need to change your image name same as your username (upama677.jpeg).
- Your image extension is .jpeg but you have written .jpg.